doc/Analytics.xml

```diff
@@ -3049,6 +3049,480 @@
           </variablelist>
         </section>
       </section>
+    </section>
+	<section xml:id="mqtt">
+      <title>Publishing ONVIF metadata to an MQTT broker</title>
+
+      <para>Increasing demand in analytics use cases presented a need for
+      standardizing format and transport of ONVIF metadata to be consumed by
+      non-traditional clients (e.g. cloud vendors) to bridge gap between
+      traditional surveillance systems and IoT ecosystems. For easy
+      consumption in such systems ONVIF metadata can be published in JSON
+      format over the MQTT protocol.</para>
+
+      <para>MQTT is a pub/sub messaging protocol centered around topics.
+      Consumers can subscribe to topics published by producers to get
+      continuous updates on the data content. ONVIF metadata, like video
+      analytics scene descriptions, can be published to an MQTT broker and
+      subscribed to by a consumer.</para>
+
+      <para>This is the process for setting up the data flow:<itemizedlist>
+          <listitem>
+            <para>An ONVIF client calls AddEventBroker in the device's Event
+            service to set up:<itemizedlist>
+                <listitem>
+                  <para>Address and credentials to the broker to use</para>
+                </listitem>
+
+                <listitem>
+                  <para>A unique TopicPrefix for the device</para>
+                </listitem>
+
+                <listitem>
+                  <para>A PublishFilter listing which topics that the device
+                  shall publish</para>
+                </listitem>
+              </itemizedlist></para>
+          </listitem>
+
+          <listitem>
+            <para>The producer connects to the broker and starts publishing
+            data on the selected topics.</para>
+          </listitem>
+
+          <listitem>
+            <para>The consumer connects to the broker, specifying which topics
+            that it wants to subscribe to. A consumer can be any MQTT capable
+            client, for example, an event engine like Node-RED, IoT cloud
+            platforms, VSaaS platforms, etc. Multiple comsumers using
+            different topics are possible.</para>
+          </listitem>
+
+          <listitem>
+            <para>The consumer receives updates on subscribed topics and can
+            implement actions like statistics aggregation and complex
+            analytics rules with data from multiple sources.</para>
+          </listitem>
+        </itemizedlist></para>
+
+      <para>The following sections describe how topics and payload shall be
```

Comment:
This informative section should go into overview. Note that the current overview is very short and hence would need a bigger rewording.